### ISSUE TITLE
performance, unnecessary function calls

### ISSUE THREAD

**Issue by quantenschaum** [aendie/SkyAlmanac-Py3#4]
I found this in [several places](https://github.com/aendie/SkyAlmanac-Py3/blob/master/alma_skyfield.py#L462)

```py
    ra = position.apparent().radec(epoch='date')[0]
    dec = position.apparent().radec(epoch='date')[1]
```

This performs the apparent and radec calculations twice. Better write

```py
ra, dec, _ = position.apparent().radec('date')
```

**Comment by aendie** [comment]
Absolutely correct. Thanks for the tip - I'll correct it in the next update.
I have my head deep into Planetary Phenomena currently ;-)

**Comment by quantenschaum** [comment]
Nice. I also had a take on generating a Nautical Almanac. You may want to have a look into https://github.com/quantenschaum/nautical_almanac

**Comment by aendie** [comment]
I have now published sfalmanac & skyalmanac 1.11.5. They're the same program now, by the way, so it doesn't matter which you use.

Your suggestion above is implemented. A few other fixes/enhancements are also included.

If you have no other problems, I'll close this issue after a few days.

**Comment by quantenschaum** [comment]
Nice, sure close it, it was more a remark than an actual bug.
